src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -333,7 +333,7 @@ private enum IsParamsValues : byte
                         Debug.Assert(refKind != RefKind.None);
                         scope = ScopedKind.ScopedRef;
```

Comment:
Can we have `T: allows ref struct` parameter symbol which is also byref, in metadata?

Reply:
> Can we have `T: allows ref struct` parameter symbol which is also byref, in metadata?

We can have one in source as well. There is nothing wrong with that.